README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 4.10.2
```

Comment:
The code change from "testing semantic versioning 4.10.2" to "testing semantic versioning 4.10.7" seems correct as it indicates a version update. No further suggestions needed.